site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -16,16 +16,18 @@ npx cap sync
 Similar to the installation steps. Capacitor will detect the new version of the plugin and install it.
 
 ```bash
-npm install cordova-plugin-name
+npm install cordova-plugin-name@latest
 npx cap update
 ```
 
+If you don't want to risk to introduce breaking changes, use `npm update cordova-plugin-name` instead of `@latest`.
```

Comment:
maybe say why update command doesn't introduce breaking changes, like "use `npm update cordova-plugin-name` instead of `@latest` as `update` respects semver".